ui/apps/platform/cypress/integration/accessControl/accessControlRoles.test.js

```diff
@@ -1,75 +1,56 @@
-import { rolesUrl, selectors } from '../../constants/AccessControlPage';
-import { permissions as permissionsApi } from '../../constants/apiEndpoints';
-
 import withAuth from '../../helpers/basicAuth';
-import { getRegExpForTitleWithBranding } from '../../helpers/title';
-
-// Migration from cy.server and cy.route to cy.intercept fails for /v1/roles/* imported from apiEndpoints.
-const rolesApi = {
-    list: '/v1/roles',
-};
 
-const h1 = 'Access Control';
-const h2 = 'Roles';
+import {
+    assertAccessControlEntityDoesNotExist,
+    clickEntityNameInTable,
+    rolesKey as entitiesKey,
```

Comment:
Nit: why the alias for these keys? Reading below I see
```typescript
        visitAccessControlEntities(entitiesKey);
```
which at first glance looks like `entitiesKey` is variable somewhere else in the file. Without the alias it would be more clear which entity we are visiting.
```typescript
        visitAccessControlEntities(rolesKey);
```
At least if the pattern to only visit one access control entity per test file is the plan for the future.

Reply:
Good question. First impression from reader viewpoint is valuable to know.

Although test files for these “entities” have more differences than for workflow entities, some tests are similar:
* `'displays alert if no permission'`
* `'list has table head cells'`
* `'displays message instead of form if entity id does not exist'`

The pro and con are worth consideration whether during copy-paste of tests to edit:
* only what is **different** for an entity
* also distinguish **which** entity: `accessScopesKey` versus `authProvidersKey` versus `permissionSetsKey` versus `rolesKey`